doc/netplan.md

```diff
@@ -572,6 +580,31 @@ These options are available for all types of interfaces.
           see ``/etc/iproute2/rt_tables``.
           (``NetworkManager``: as of v1.10.0)
 
+    ``keepalive`` (scalar) – since **0.100**
+    :    An interval in seconds, between 1 and 65535 inclusive, of how often to
+         send an authenticated empty packet to the Wireguard peer for the
+         purpose of keeping a stateful firewall or NAT mapping valid
+         persistently. Optional.
+
+    ``keys`` (mapping) – since **0.100**
+    :    Define keys to use for the routes, specifically the Wireguard peers.
+
+         This field can be used as a mapping, where you can further specify the
+         ``public`` and ``shared`` keys.
+
+         ``public`` (scalar) – since **0.100**
+         :    A base64-encoded public key, requried for Wireguard peers.
+
+         ``shared`` (scalar) – since **0.100**
+         :    A base64-encoded preshared key. Optional for Wireguard peers.
+              When the ``systemd-networkd`` backend (v242+) is used, this can
+              also be an absolute path to a file containing the preshared key.
+
+     ``key`` (scalar) – since **0.100**
```

Comment:
I would prefer not to see alternate schema options for expressing the same content.

Reply:
Okay. I removed `key` setting in favor of just `keys` within wireguard-peers.
We will still have the tunnel `key[s]` for historical reason (I don't want to break the backwards compatibility there).